crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -37,8 +38,8 @@ pub struct StakeTableEntryVar {
     pub stake_amount: Variable,
 }
 
-/// HotShot state Variable
-/// The stake table commitment is a triple (bls_keys_comm, stake_amount_comm, schnorr_keys_comm).
+/// Light client state Variable
+/// The stake table commitment is a triple (bls_keys_comm, schnorr_keys_comm, stake_amount_comm).
 #[derive(Clone, Debug)]
 pub struct LightClientStateVar {
```

Comment:
Can we define internal `struct StakeTableCommitmentVar` to avoid the `.0, .1, .2` access? 
we could have `stake_table_comm_var.bls_key_comm()` instead.

Reply:
check https://github.com/EspressoSystems/HotShot/commit/599a75df2a83eeddfadb783d23cfccb30ad2d494